docs/Example.vue

```diff
@@ -33,6 +33,10 @@
       </sui-menu>
     </sui-grid-column>
 
+    <sui-grid-column v-if="info">
```

Comment:
Adding info message like this https://semantic-ui.com/modules/embed.html#custom-content

Reply:
Great!